content/posts/newsletter/2019-10-01.md

```diff
@@ -0,0 +1,124 @@
+
+![Monal and Airdrop screenshots](https://framapic.org/Qg1mv43w1673/akPmtepp0nfH.png)
+
+JabberFr, the French XMPP/Jabber community, has again [translated the XMPP newsletter in French](https://news.jabberfr.org/2019/09/lettre-dactualite-xmpp-du-3-septembre-2019/), merci beaucoup.
+
+We have started a very minimalistic [communication guide](https://wiki.xmpp.org/web/Basic_communication_guide_for_XMPP_techies) to help promoting a project through social networks and other means such as blog posts. Could be worth for those involved in XMPP projects and would like to get ideas on how to reach out to different communities.
+
+## Events
+
+As usual every year, that time has come: we proudly announce the XMPP Summit 24 and [FOSDEM](https://fosdem.org/2020/). The 24th XMPP Summit will happen on Thursday 30th and Friday 31st of January, and FOSDEM will be held on Saturday 1st and Sunday 2nd of February. Prepare for the community gathering; now is good time to start booking your flights!
+
+![FOSDEM 2020](https://framapic.org/fTZlikXsyi0F/ZqlvJhNjWDFj.png)
+
+## Software releases
+          
+### Servers
+
+Réda Housni Alaoui is reviving the formerly dormant [Vysper XMPP server](https://github.com/apache/mina-vysper) (pronounced as "whisper").
+
+
+[Xabber Server v.0.9 alpha is released](https://blog.xabber.com/xabber-server-v-0-9-alpha-is-released/), with quick installation and management panel and interesting innovations.
+
+[MongooseIM 3.4.1 is out now](https://www.erlang-solutions.com/resources/download.html) with an important security upgrade, fixing a vulnerability allowed any logged in user to crash the node with malicious stanza on certain (but popular) configurations. [Read the whole thread on Twitter](https://twitter.com/MongooseIM/status/1176111308430815232) for more information.
+
+The igniterealtime community has many news:
+* [Openfire 4.4.2](https://discourse.igniterealtime.org/t/openfire-4-4-2-release/86209): "This release should better support server to server (s2s) connections, fix a few admin console XSS-style issues, and improve client session stability."
+* The Fastpath Service plugin has been released in versions [4.4.4](https://discourse.igniterealtime.org/t/fastpath-service-plugin-4-4-4-released/86155) and [4.4.5](https://discourse.igniterealtime.org/t/fastpath-service-plugin-4-4-5-released/86198), bringing support for managed queued chat requests, such as a support team might use.
+* [Search plugin 1.7.3](https://discourse.igniterealtime.org/t/search-plugin-1-7-3-released/86207): "This update 3 adds protection against CSRF and XSS attacks."
+* [Monitoring Service plugin 1.8.1](https://discourse.igniterealtime.org/t/monitoring-service-plugin-1-8-1-released/86208): "This hotfix update 4 adds protection against XSS attacks on Archiving Settings page."
```

Comment:
"This hotfix update, adds protection" (adding a comma and removing the 4)

Reply:
not even a comma, I think